src/main/java/org/sonarqube/A.java

```diff
@@ -1,4 +1,14 @@
 package org.sonarqube;
 
 public class A {
+  String message = "foo";
+
+  // TODO fix this
```

Comment:
###AWADjq7FTf3tEBk_CZ7Z###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_3@6
Complete the task associated to this TODO comment.